content/kubermatic/main/installation/start-kkp/guides/wizard/_index.en.md

```diff
@@ -0,0 +1,109 @@
++++
```

Comment:
I think this file shouldn't be here? I suspect it existed when this PR was started but removed in the mean time. Can you remove it again please?